openstudiocore/src/gbxml/Test/ReverseTranslator_GTest.cpp

```diff
@@ -93,6 +100,302 @@ TEST_F(gbXMLFixture, ReverseTranslator_ZNETH)
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("8 Space", space->name().get());
+
+  osurf = model->getModelObjectByName<Surface>("T-4-8-I-F-20 Reversed");
+  ASSERT_TRUE(osurf);
+  EXPECT_EQ("RoofCeiling", osurf->surfaceType());
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("4 Space", space->name().get());
+
+  osurf = model->getModelObjectByName<Surface>("T-5-9-I-F-25");
+  ASSERT_TRUE(osurf);
+  EXPECT_EQ("Floor", osurf->surfaceType());
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("9 Space", space->name().get());
+
+  osurf = model->getModelObjectByName<Surface>("T-5-9-I-F-25 Reversed");
+  ASSERT_TRUE(osurf);
+  EXPECT_EQ("RoofCeiling", osurf->surfaceType());
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("5 Space", space->name().get());
+
+  osurf = model->getModelObjectByName<Surface>("T-6-10-I-F-29");
+  ASSERT_TRUE(osurf);
+  EXPECT_EQ("Floor", osurf->surfaceType());
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("10 Space", space->name().get());
+
+  osurf = model->getModelObjectByName<Surface>("T-6-10-I-F-29 Reversed");
+  ASSERT_TRUE(osurf);
+  EXPECT_EQ("RoofCeiling", osurf->surfaceType());
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("6 Space", space->name().get());
+
+  osurf = model->getModelObjectByName<Surface>("T-7-11-I-F-33");
+  ASSERT_TRUE(osurf);
+  EXPECT_EQ("Floor", osurf->surfaceType());
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("11 Space", space->name().get());
+
+  osurf = model->getModelObjectByName<Surface>("T-7-11-I-F-33 Reversed");
+  ASSERT_TRUE(osurf);
+  EXPECT_EQ("RoofCeiling", osurf->surfaceType());
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("7 Space", space->name().get());
+
+  osurf = model->getModelObjectByName<Surface>("T-8-12-I-F-36");
+  ASSERT_TRUE(osurf);
+  EXPECT_EQ("Floor", osurf->surfaceType());
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("12 Space", space->name().get());
+
+  osurf = model->getModelObjectByName<Surface>("T-8-12-I-F-36 Reversed");
+  ASSERT_TRUE(osurf);
+  EXPECT_EQ("RoofCeiling", osurf->surfaceType());
+  space = osurf->space();
+  ASSERT_TRUE(space);
+  EXPECT_EQ("8 Space", space->name().get());
+
+}
+
+TEST_F(gbXMLFixture, ReverseTranslator_AlternateUnits)
+{
+  openstudio::path inputPath = resourcesPath() / openstudio::toPath("gbxml/TestCubeAlternateUnits.xml");
+
+  openstudio::gbxml::ReverseTranslator reverseTranslator;
+  boost::optional<openstudio::model::Model> model = reverseTranslator.loadModel(inputPath);
+  ASSERT_TRUE(model);
+
+  auto surfs = model->getModelObjects<Surface>();
+
+  OptionalSurface osurf = model->getModelObjectByName<Surface>("T-1-5-I-F-6");
+  ASSERT_TRUE(osurf);
+  auto points = osurf->vertices();
+  ASSERT_EQ(4, points.size());
+  EXPECT_EQ(0.9144, points[1].y());
+
+  //auto omat = model->getModelObjectByName<StandardOpaqueMaterial>("Concrete: 100 [mm]");
```

Comment:
@macumber Once someone can get the unit conversion stuff to work, then the "alternate units" test file would allow these to be uncommented (if I did the manual conversions correctly). Right now these will all fail because the none of the material properties get converted and SI is assumed. So it is waiting on more fixes.